CMakeLists.txt

```diff
@@ -338,19 +338,19 @@ if(NOT "${BASH}" STREQUAL "BASH-NOTFOUND" AND GINKGO_DEVEL_TOOLS)
     find_program(GIT git)
     if(NOT "${GIT}" STREQUAL "GIT-NOTFOUND")
         add_custom_target(format_header
-            COMMAND echo "format header on the modified code files except build/examples/third_party/ginkgo.hpp"
+            COMMAND echo "format header on the modified code files except build/examples/third_party/accessor/dev_tools/ginkgo.hpp"
             COMMAND bash -c "git diff --name-only origin/develop...HEAD | \
-                grep -Ev 'build|examples|third_party|ginkgo.hpp' | \
+                grep -Ev 'build|examples|third_party|accessor/|dev_tools/|ginkgo.hpp' | \
                 grep -E '(\.hip)?\.(cu|hpp|cuh|cpp)$' | \
                 xargs -r -n1 ${Ginkgo_SOURCE_DIR}/dev_tools/scripts/format_header.sh"
             WORKING_DIRECTORY ${Ginkgo_SOURCE_DIR}
             VERBATIM)
     endif()
     unset(GIT CACHE)
     add_custom_target(format_header_all
-        COMMAND echo "format header on all code files except build/examples/third_party/ginkgo.hpp"
+        COMMAND echo "format header on all code files except build/examples/third_party/accessor/dev_tools/ginkgo.hpp"
         COMMAND bash -c "find * -type f | \
-                grep -Ev 'build|examples|third_party|ginkgo.hpp' | \
+                grep -Ev 'build|examples|third_party|accessor/|dev_tools/|ginkgo.hpp' | \
```

Comment:
same as above
```suggestion
                grep -Ev 'build|examples|third_party|accessor|dev_tools|ginkgo.hpp' | \
```

Reply:
I think we have headers whose name contains `accessor`, so the former is definitely useful

Reply:
Yes, some name contains accessor, so I only remove the accessor folder
dev_tools can be without `/`